support.php

```diff
@@ -16,21 +16,30 @@
     $guest_support = false;
     $player = User::loadFromId($system, $_SESSION['user_id']);
     $player->loadData();
-    $layout = $system->setLayoutByName($player->layout);
+    $system->setLayoutByName($player->layout);
     $staff_level = $player->staff_level;
     $user_id = $player->user_id;
 
+    $render_sidebar_header = true;
     $supportSystem = new SupportManager($system, $player);
 }
 else {
-    $layout = $system->setLayoutByName("shadow_ribbon");
+    $render_sidebar_header = false;
```

Comment:
What this ends up representing is "is the player logged in, or not". That's information the `renderBeforeContentHTML` function already has (is player null or not), could we tie the logic you're thinking of to that?